kadai.js

```diff
@@ -0,0 +1,67 @@
+window.onload = function () {
+    const textArea = document.getElementById('textarea');
+    const errMessage = () => {
+        var p = document.createElement('p');
+        p.textContent = '整数値を入力してください';
+        textArea.appendChild(p);
+    }
+    const writeTest = (t) => {
+        var p = document.createElement('p');
+        p.textContent = t;
+        textArea.appendChild(p);
+    }
+    document.getElementById("btn").onclick = () => {
+        while (textArea.firstChild) {
+            textArea.removeChild(textArea.firstChild );
+        }
+        const numList = [];
+        const inputF = document.getElementById('fizz').value;
+        const input_fizz = Number(inputF);
+        const inputB = document.getElementById('buzz').value;
+        const input_buzz = Number(inputB);
+        if (input_fizz === 0) {
+            errMessage();
```

Comment:
このように書くと if 文がネストしてしまうので、以下のようにするとネストのレベルが減らせます。
```suggestion
            errMessage();
            return;
```